test/e2e/kubectl/kubectl.go

```diff
@@ -284,7 +284,7 @@ var _ = SIGDescribe("Kubectl client", func() {
 			Testname: Kubectl, replication controller
 			Description: Create a Pod and a container with a given image. Configure replication controller to run 2 replicas. The number of running instances of the Pod MUST equal the number of replicas set on the replication controller which is 2.
 		*/
-		framework.ConformanceIt("should create and stop a replication controller ", func() {
+		framework.ConformanceIt("should create and stop a replication controller [Arktos-CI]", func() {
```

Comment:
Do we have a test for replicaset controller or deployment controller? Replication Controller is deprecated

Reply:
Currently I'm adding them. I'm using the preset machines to verify.

Reply:
This is a test for kubectl. Try not use deprecated resource. It feels like "Kubectl run deployment" would be a better core test.

Reply:
I switched to another case. Will show in next iteration.